client/src/app/components/nova/address/section/nft/Nft.tsx

```diff
@@ -56,6 +65,10 @@ const Nft: React.FC<NftProps> = ({ nftOutput }) => {
             if (metadata) {
                 setIssuerId(nftIssuerId);
             }
+            const nftId = nftOutput.nftId === ZEROED_ID ? Utils.computeNftId(outputId) : nftOutput.nftId;
```

Comment:
Could we maybe use the same approach as we have in stardust just so the code is similar and we can maintain both networks with the same logic? `const nftId = HexHelper.toBigInt256(nftOutput.nftId).eq(bigInt.zero) ? Utils.computeNftId(outputId) : nftOutput.nftId;`